src/stan/variational/base_family.hpp

```diff
@@ -31,6 +31,8 @@ namespace stan {
       Eigen::VectorXd transform(const Eigen::VectorXd& eta) const;
       template <class BaseRNG>
       void sample(BaseRNG& rng, Eigen::VectorXd& eta) const;
+      template <class BaseRNG>
```

Comment:
Please add documentiation. At the very least, add documentation for every function that you're adding. Please follow these guidelines:
https://github.com/stan-dev/stan/wiki/How-to-Write-Doxygen-Doc-Comments

Reply:
documented in normal_meanfield.hpp and normal_fullrank.hpp